src/Core/TestInterface.php

```diff
@@ -90,6 +90,13 @@ public function getPending();
     public function setPending($state);
 
     /**
+     * Return whether or not the test is focused
+     *
+     * @return bool
+     */
+    public function getFocused();
```

Comment:
I remember there being some reasoning to `getPending()` . I think I remember it having something to do with pending being one of three states instead of just true/false.

I think the original thought was that null means a pending state does not exist, where true/false are explicit pending states. Hence `Suite.php` has the following:

```php
    protected function runTest(TestInterface $test, TestResult $result)
    {
        if ($this->getPending() !== null) {
            $test->setPending($this->getPending());
        }

        $test->setEventEmitter($this->eventEmitter);
        $test->run($result);
    }
```

Looking at it now, I can't remember why the explicit setting was favored....